frontend/src/App.tsx

```diff
@@ -1,35 +1,16 @@
 import React from 'react';
-import { Layout, Menu } from 'antd';
 import { Provider } from 'mobx-react';
 
 import { AppBody } from './app/app-body';
 import { rootStore } from './stores';
```

Comment:
You might notice that I split the imports up with a single line as a gap. The pattern i'm looking to keep is:
1. Imports from third party libraries (i.e. those that may appear in package.json)
2. Imports from within our code base
3. imports from CSS files

See if there is anywhere else in this review that this is violated. Keeping style consistent is important in this line of work, so we avoid unnecessary diff comparisons in code reviews